LEGO1/legovideomanager.h

```diff
@@ -33,20 +33,42 @@ class LegoVideoManager : public MxVideoManager {
 		this->m_videoParam.GetPalette()->SetOverrideSkyColor(p_shouldOverride);
 	}
 
+	virtual void Destroy() override; // vtable+0x18
+
 private:
```

Comment:
Could you add offset comments to all of the members? This will be helpful since it's a pretty big class. Thank you!